src/pages/workspace/reimburse/WorkspaceRateAndUnitPage/InitialPage.tsx

```diff
@@ -159,7 +157,5 @@ export default compose(
         workspaceRateAndUnit: {
             key: ONYXKEYS.WORKSPACE_RATE_AND_UNIT,
         },
-    }),
-    withPolicy,
-    withNetwork(),
```

Comment:
Was this not needed in the page?

Reply:
Yes its not used anywhere as specified in the props types:

https://github.com/Expensify/App/blob/4400aca0b2948f8a2945e16335975152851df5b6/src/pages/workspace/reimburse/WorkspaceRateAndUnitPage/InitialPage.tsx#L28-L31

also i forgot to remove this type, i just removed it.

Reply:
actually, as i mentioned here https://github.com/Expensify/App/pull/42069#issuecomment-2158738542  the whole ( `InitialPage`, `RatePage`,`UnitPage` ) pages are not used anywhere so i am not sure if we should remove these components or not and if so should we do it in the same PR or in a new one.